modules/querying/pages/func/type-conversion-functions.adoc

```diff
@@ -244,7 +246,7 @@ Converts an integer or string to a floating-point number.
 
 * If the input is already a floating-point number, the function will return it unchanged.
 * If the input is a valid number in string format (e.g., "14.5"), it converts the string to a float (e.g., 14.5).
-* *Scientific notation* is supported. For example, `1.45E-6` will be correctly interpreted as `1.45e-06`.
+* *Scientific notation* is supported. For example, "1.45E-6" will be correctly interpreted as "1.45e-06".
```

Comment:
Actually, both the before and after versions are not quite right.
The BEFORE should be a number. 
The AFTER should be a string, with double quotation marks.

Optionally, you can put both of them in code style, to be explicit that you are showing a code example, not just regular English.